nixio/container.py

```diff
@@ -83,6 +98,55 @@ def items(self):
             yield item.id, item
 
 
+class SectionContainer(Container):
+    """
+    SectionContainer extends Container with a new __delitem__ method.
+    When a Section is deleted, all child sources need to be deleted
+    individually to make sure all their references are removed.
```

Comment:
probably child *sections* ;)

Reply:
Well fine, if you want to be all _correct_ about it